Makefile

```diff
@@ -73,9 +73,14 @@ event-handler:
 
 # Run all tests
 .PHONY: test
-test:
+test: test-tooling
 	@echo Testing plugins against Teleport $(TELEPORT_GET_VERSION)
-	go test -race -count 1 ./...
+	go test -race -count 1 $(shell go list ./...)
```

Comment:
Not sure if my comment is applicable here, but looks like. Terraform provider tests need env var `TF_ENV=true` to be specified (Terraform framework requirement). Terraform must be available as well.

Reply:
This is change is to avoid `go test` picking up the `tooling` module here. For some reason, in my testing, `./...` included the `tooling` package and failed, where `$(shell go list ./...)` correctly treated it as a separate module.

Just tested it again now an all seems fine.